app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -191,8 +189,15 @@ impl View {
     /// Constructor.
     pub fn new(app: &Application) -> Self {
         let frp = Frp::new();
-        let model = Model::new(app, frp.open_project.clone_ref(), frp.create_project.clone_ref());
         let network = &frp.network;
+        frp::extend! { network
+            def open_project = source();
+            def create_project = source();
```

Comment:
A more common version is:
```suggestion
            open_project <- source();
            create_project <- source();
```

Reply:
Thank you, good to know!